sdk/core/Azure.Core/src/Pipeline/TelemetryPolicy.cs

```diff
@@ -2,10 +2,11 @@
 // Licensed under the MIT License.
 
 using System.Runtime.InteropServices;
+using Azure.Core.Http;
 
-namespace Azure.Core.Pipeline.Policies
+namespace Azure.Core.Pipeline
 {
-    public class TelemetryPolicy : SynchronousHttpPipelinePolicy
+    internal class TelemetryPolicy : SynchronousHttpPipelinePolicy
```

Comment:
Was this intentional?  It's the only non-namespace change.

Reply:
Yeah, I noticed that I forgot TelemetryPolicy when making build-in policies internal.